api4/config.go

```diff
@@ -407,6 +407,12 @@ func migrateConfig(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	isCloud := c.App.License() != nil && *c.App.License().Features.Cloud
```

Comment:
Can we make this consistent with other places: 
`c.App.Srv().License() != nil && *c.App.Srv().License().Features.Cloud`